--- xreader-2.8.3/meson.build.orig	2023-06-16 19:13:12.439883975 +0200
+++ xreader-2.8.3/meson.build	2023-06-16 19:46:56.882249963 +0200
@@ -71,7 +71,7 @@ if get_option('ps')
     xreader_mime_types += 'application/postscript;application/x-bzpostscript;application/x-gzpostscript;image/x-eps;image/x-bzeps;image/x-gzeps;'
 endif
 if get_option('dvi')
-    kpathsea = dependency('kpathsea')
+    kpathsea = cc.find_library('kpathsea')
     spectre = dependency('libspectre', version: '>= 0.2.0')
     if get_option('t1lib')
         t1lib = cc.find_library('t1', required: false)
